### PR TITLE
Zombie cure bottles / injectors for admins, no synth cures, updates comments

### PR DESCRIPTION
**Pull request by Qwertytoforty**
## What Does This PR Do

Zombie cure effects comments updated.
Adds 4 zombie cure bottles for admins to spawn.
Adds 4 zombie cure autoinjectors for admins to spawn. These can not be drained out of, so you can give them to crew without them being able to duplicate it.
Zombie cures are no longer no_synth. Sorry bee cure methods, you will be missed.

## Why It's Good For The Game

Accurate comments good.

Being able as admins to easily spawn the cure bottles good.
Being able as admins to spawn autoinjectors, be it as loot, ERT, balancing, preperation, or other is good, without letting crew cheese cures out of it.
Blocks the chemicals from being duplicated by ody / bees, as part of the above. Sorry.

## Testing

Spawning them, checking they had the right reagents

## Changelog
:cl:
tweak: Admins can now directly spawn vials of the zombie cures.
tweak: Admins can now spawn injectors of the 4 zombie cures. Players can *not* duplicate a virus cure out of these, and are safe to give out without them being able to duplicate it.
tweak: Zombie virus cures can no longer be duplicated by bees or ody mechs.
/:cl:

### CODE REVIEW

File: code/modules/reagents/reagent_containers/hypospray.dm [hunk 1]
```diff
@@ -224,3 +224,31 @@
...
+	desc = "A small autoinjector containing 15, units of Anti-Plague Sequence Alpha. Prevents infection, cures level 1 infection."
+	icon_state = "zombiepen"
+	amount_per_transfer_from_this = 15
+	volume = 15
+	container_type = null //No sucking out the reagent
+	list_reagents = list("zombiecure1" = 15)
+
+/obj/item/reagent_containers/hypospray/autoinjector/zombiecure/attack(mob/living/M, mob/user)
+	if(..())
+		playsound(loc, 'sound/weapons/smg_empty_alarm.ogg', 20, 1) //Sucker for sounds, also gets zombies attention.
```
Comment (Sirryan2002) [inline code/modules/reagents/reagent_containers/hypospray.dm:239]:
```suggestion
		playsound(loc, 'sound/weapons/smg_empty_alarm.ogg', 20, TRUE) //Sucker for sounds, also gets zombies attention.
```

**Review by Sirryan2002 — APPROVED** [review]

File: code/modules/reagents/reagent_containers/hypospray.dm [hunk 2]
```diff
@@ -224,3 +224,31 @@
 /obj/item/reagent_containers/hypospray/autoinjector/nanocalcium/attack(mob/living/M, mob/user)
 	if(..())
 		playsound(loc, 'sound/weapons/smg_empty_alarm.ogg', 20, 1)
+
+/obj/item/reagent_containers/hypospray/autoinjector/zombiecure
+	name = "\improper Anti-Plague Sequence Alpha autoinjector"
+	desc = "A small autoinjector containing 15, units of Anti-Plague Sequence Alpha. Prevents infection, cures level 1 infection."
```
Comment (lewcc) [inline code/modules/reagents/reagent_containers/hypospray.dm:230]:
```suggestion
	desc = "A small autoinjector containing 15 units of Anti-Plague Sequence Alpha. Prevents infection, cures level 1 infection."
```

File: code/modules/reagents/reagent_containers/hypospray.dm [hunk 3]
```diff
@@ -224,3 +224,31 @@
...
+	container_type = null //No sucking out the reagent
+	list_reagents = list("zombiecure1" = 15)
+
+/obj/item/reagent_containers/hypospray/autoinjector/zombiecure/attack(mob/living/M, mob/user)
+	if(..())
+		playsound(loc, 'sound/weapons/smg_empty_alarm.ogg', 20, 1) //Sucker for sounds, also gets zombies attention.
+
+/obj/item/reagent_containers/hypospray/autoinjector/zombiecure/zombiecure2
+	name = "\improper Anti-Plague Sequence Beta autoinjector"
+	desc = "A small autoinjector containing 15, units of Anti-Plague Sequence Beta. Weakens zombies, heals low infections."
```
Comment (lewcc) [inline code/modules/reagents/reagent_containers/hypospray.dm:243]: Fix all the commas here

**Review by lewcc — CHANGES_REQUESTED** [review]

**Review by lewcc — APPROVED** [review]
lgt--braaaains

**Pull request merged**